aiida_vasp/utils/aiida_utils.py

```diff
@@ -87,6 +89,71 @@ def builder_interface(calc_cls):
     return False
 
 
+def new_structure(old_structure):
+    """Assemble a new StructureData."""
+    structure_cls = get_data_class('structure')
+    old_cell = old_structure.cell
+    structure = structure_cls(cell=old_cell)
+    old_sites = old_structure.sites
+    kinds = old_structure.kinds
+    for index, site in enumerate(old_sites):
+        structure.append_atom(position=site.position, symbols=kinds[index].symbols, name=kinds[index].name)
+    return structure
+
+
+def new_parameter(old_parameter):
+    """Assemble a new ParameterData."""
+    parameter_cls = get_data_class('parameter')
```

Comment:
simplify to `return get_parameter_node('parameter', dict=old_parameter.get_dict())`

Reply:
also later instances

Reply:
Fixed